raiden/transfer/state.py

```diff
@@ -76,6 +76,23 @@ def message_identifier_from_prng(prng):
     return prng.randint(0, UINT64_MAX)
 
 
+InitiatorTask = namedtuple('InitiatorTask', (
```

Comment:
Since you are making this change can you use the typed version of the named tuple? https://docs.python.org/3/library/typing.html#typing.NamedTuple